### PR TITLE
fixed issue #1476 scroll-behavior example problem

### PR DESCRIPTION
**Pull request by ghost**
Height of iframe was inappropriate
Height of iframe was 250px which was too less so I updated height to 450px.
issue was #1476

### CODE REVIEW

File: files/en-us/web/css/scroll-behavior/index.html [hunk 1]
```diff
@@ -103,7 +103,7 @@ <h4 id="CSS">CSS</h4>
 
 <h4 id="Result">Result</h4>
 
-<p>{{ EmbedLiveSample("Setting_smooth_scroll_behavior", "100%", 250) }}</p>
```
Comment (peterbe) [inline files/en-us/web/css/scroll-behavior/index.html:106]: Shouldn't it be `450` (instead of `"450px"`)? Did you test it?

**Comment by rachelandrew** [comment]
@peterbe this is this problem I believe https://github.com/mdn/yari/issues/2683 so changing the height will not make any difference - it's mentioned in the issue as well.

I suggest we close this.

File: files/en-us/web/css/scroll-behavior/index.html (hunk 1 above)
Comment (Aayushap1) [reply files/en-us/web/css/scroll-behavior/index.html:106]: done sir

**Comment by rachelandrew** [comment]
This is part of a wider issue as explained above and won't help in this case, so I'm closing the PR.

@Aayushap1 as has been explained elsewhere, you do need to get setup to test locally before submitting pull requests.

**Pull request closed without merging**